Doc/library/token-list.inc

```diff
@@ -203,6 +203,10 @@
 
 .. data:: OP
 
+.. data:: TYPE_IGNORE
+
+.. data:: TYPE_COMMENT
+
```

Comment:
I would also update `token.rst`, there are few sentences about `COMMENT`, I think it would be good to update them.

Reply:
Done.